src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -165,13 +167,24 @@ export default {
     disableAdaptiveLoadingInput() {
       this.well.loading_target_p1_plus_p2 = ''
     },
-    async showModalForPosition() {
+    async showModalForPosition(position) {
+      if (position) {
+        this.positionData = position
+      }
       // We also need to setup the well here in case state has updated since
       await this.setupWell()
-      this.$refs['well-modal'].show()
+      this.isShow = true
+      /**This need to be removed when custom_enable_modal feature flag is removed */
+      if ('b-modal' in this.$refs['well-modal'].$refs) {
```

Comment:
This looks almost the same as the hide method. Maybe it could be combined?

Reply:
Thought it would be better to have it separate as one is show and other hide. Also, 'showModalForPosition' is an async operation called from child component and doing some extra things _if (position) {
        this.positionData = position
      }
      // We also need to setup the well here in case state has updated since
      await this.setupWell()_
Also, thought it would be easy to support binding bootstrap events. Please share your thoughts